dffml/source/file.py

```diff
@@ -49,6 +52,16 @@ def __repr__(self):
         elif self.filename[::-1].startswith(('.xz')[::-1]) or \
                 self.filename[::-1].startswith(('.lzma')[::-1]):
             opener = lzma.open(self.filename, 'rt')
+        elif self.filename[::-1].startswith(('.zip')[::-1]):
+            @contextmanager
+            def opener_helper():
+                with zipfile.ZipFile(self.filename) as archive:
+                    for file in archive.infolist():
```

Comment:
`file` is a built-in, please use `zipfilename` instead (or anything that's not `file`)

Reply:
Done :+1: